### PR TITLE
Translation for Nikkei article

### PR DESCRIPTION
**Pull request by agektmr**
Translation for Nikkei article

[Check out a review request at GitLocalize](https://gitlocalize.com/repo/107/ja/review/7747)

### CODE REVIEW

**Comment by agektmr** [comment]
@studiomohawk Can you sign the CLA?

File: src/content/ja/showcase/2018/nikkei.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+{# wf_blink_components: N/A #}
+{# wf_featured_image: /web/showcase/2018/images/nikkei/featured.jpg #}
+{# wf_featured_snippet: 日本経済新聞は140年を超える歴史を持つ、日本において信頼されるメディアの1つです。より優れたユーザ体験とWebにおけるビジ値すっを加速するため、日本経済新聞社は2017年11月、プログレッシブウェブアプリをローンチし、新たなプラットフォームで優れた結果を出すことができました。 #}
+{# wf_tags: casestudy,progressive-web-apps #}
+{# wf_region: asia #}
+{# wf_vertical: media #}
+{# wf_featured_date: 2018-11-19 #}
+{# wf_blink_components: N/A #}
+
+# 日本経済新聞社は複数ページのPWAで新しいレベルの品質とパフォーマンスを実現  {：.page-title}
```
Comment (agektmr) [inline src/content/ja/showcase/2018/nikkei.md]: Replace "：" with ":"

**Comment by studiomohawk** [comment]
@agektmr did signed the CLA.

File: src/content/ja/showcase/2018/nikkei.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
+project_path: /web/_project.yaml
+book_path: /web/showcase/_book.yaml
+
+{# wf_published_on: 2018-11-19 #}
+{# wf_updated_on: 2018-11-19 #}
+{# wf_blink_components: N/A #}
+{# wf_featured_image: /web/showcase/2018/images/nikkei/featured.jpg #}
+{# wf_featured_snippet: 日本経済新聞は140年を超える歴史を持つ、日本において信頼されるメディアの1つです。より優れたユーザ体験とWebにおけるビジ値すっを加速するため、日本経済新聞社は2017年11月、プログレッシブウェブアプリをローンチし、新たなプラットフォームで優れた結果を出すことができました。 #}
```
Comment (studiomohawk) [inline src/content/ja/showcase/2018/nikkei.md]: Typo: `ビジ値すっ` → `ビジネス`
Should I fix this at GitLocalize?

Comment (agektmr) [reply src/content/ja/showcase/2018/nikkei.md]: I'm not sure fixing it on GitLocalize will be reflected here. Can you try it?

Comment (studiomohawk) [reply src/content/ja/showcase/2018/nikkei.md]: I did fixed it on GitLocalize now.
I'm really not sure how GitLocalize works.....
(fixed https://github.com/google/WebFundamentals/pull/7266#discussion_r257189019 as well)

**Comment by agektmr** [comment]
I could update the commit from GitLocalize. Now we need to see how we can update CLA status. Let's see...

**Comment by agektmr** [comment]
@studiomohawk Thank you for your contribution!

**Pull request merged**